### PR TITLE
Fixes: #16 - Added a issue template and Pull request templates

### PR DESCRIPTION
**Pull request by isuruAb**
Added  Issue template and Pull Request template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```
Comment (dinukadesilva) [inline .github/PULL_REQUEST_TEMPLATE:1]: Few suggestions,

**Branch name**
`gh-`<--related issue number-->

eg: `gh-34`

**Commit message title**
`gh-`<--related issue number-->: <--commit title-->

eg: `gh-34: Updated variable names`

**Commit message description**
fixes <--issue number-->
closes <--issue number-->
etc.

**Pull request title**
<--branch-name-->: <--pull request title-->

**Review by dinukadesilva — CHANGES_REQUESTED** [review]

**Comment by dinukadesilva** [comment]
@isuruAb 
Create links to these files from readme.
https://github.com/99xt/youtube-queuing-bot/issues/3 can be taken as a part of this PR

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (rehrumesh) [reply .github/PULL_REQUEST_TEMPLATE:1]: Good suggestion. However we don't follow specific rules for branching. Contributors are free to use their own format. However commit message should start with `#issue_id <description>`

**Comment by dinukadesilva** [comment]
This is nice. It reminded me to add demo links too.

image

**Comment by dinukadesilva** [comment]
How about having the titles bold.

**Pull request merged**